e2e-tests/package.json

```diff
@@ -20,9 +20,9 @@
   },
   "engines": {
     "node": ">=18",
-    "pnpm": ">=8"
+    "pnpm": ">=9"
   },
-  "packageManager": "pnpm@8.15.7",
+  "packageManager": "pnpm@9.5.0",
```

Comment:
I think we can remove this and allow toolings to use their latest PNPM 9 minor/patch versions

Reply:
yes, makes sense
thanks

https://github.com/FuelLabs/fuel-connectors/pull/224/commits/948eab1e0590c0a10097d0028ca9a6ab7e020ddb